src/components/Layout/Layout.js

```diff
@@ -18,6 +18,7 @@ import Homepage from "../../pages/homepage/Homepage.js";
 import Liquidations from "../../pages/liquidations/Liquidations.js";
 import Liquidator from "../../pages/liquidations/Liquidator.js";
 import Liquidators from "../../pages/liquidations/Liquidators.js";
+import Oracles from "../oracles.js/oracles";
```

Comment:
Wrong folder structure. Folders should not have extensions. Should be `../oracles/Oracles.js`
Also capitalisation. Look through the folder and file structure how we have it and stick to it.
Rough guidelines:
pages - folders always lowercase first letter then camelcase the rest
pages - files always uppercase first letter then camelcase the rest
components - folders always uppercase first letter then camelcase the rest
components - files always uppercase first letter then camelcase the rest